tests/operations/LoadProgram.hpp

```diff
@@ -0,0 +1,24 @@
+#include <fstream>
+#include <string>
+
+#include "cereal/include/cereal/archives/json.hpp"
+#include "cereal/include/cereal/cereal.hpp"
+
+#include "sgpl/program/Program.hpp"
+
+namespace sgpl::test {
+
+template <typename Spec>
```

Comment:
Move this to be a constructor of Program that takes a std::path. Use the filesystem extension member function to determine if it is .bin or .json.

Reply:
And do appropriate load. If it is neither, do an emp::error

Reply:
Also, add std::string constructor for Program that reads in JSON content